Source/WebCore/platform/graphics/texmap/TextureMapperGL.cpp

```diff
@@ -222,6 +222,7 @@ void TextureMapperGL::beginPainting(PaintFlags flags, BitmapTexture* surface)
 
 void TextureMapperGL::endPainting()
 {
+    glBindFramebuffer(GL_FRAMEBUFFER, data().targetFrameBuffer);
```

Comment:
I can't say I understand why this is needed.

Reply:
Added more commit message.